### ISSUE TITLE
Garbage generated the first time a .org file is opened

### ISSUE THREAD

**Issue by Atrifex** [syl20bnr/spacemacs#7674]
The first time I open a .org file, "te : CHANGED outside Customize. Non-nil means insert state change notes and time s" is pasted at some random location within the file. I can get rid of it by undoing a couple of times but I think that this is something that we should definitely try to fix. 

When I start spacemacs, there are no relevant warnings and these are the only errors that are generated when I *open the file*:
```
[yas] Check your `yas-snippet-dirs': ~/.spacemacs.d/snippets/ is not a directory
[yas] Reloaded everything (snippets will load just-in-time)... (some errors, check *Messages*).
[yas] Check your `yas-snippet-dirs': ~/.spacemacs.d/snippets/ is not a directory
[yas] Reloaded everything (snippets will load just-in-time)... (some errors, check *Messages*).
```

Originally File:
<img width="306" alt="screen shot 2016-11-06 at 1 45 16 pm" src="https://cloud.githubusercontent.com/assets/12935724/20040993/5b9ea110-a427-11e6-88b0-05b078980720.png">

File after Opening:
<img width="574" alt="screen shot 2016-11-06 at 1 45 08 pm" src="https://cloud.githubusercontent.com/assets/12935724/20040995/662701cc-a427-11e6-862e-a3adcfcfd853.png">

**Comment by LemmingAvalanche** [comment]
@Atrifex Steps to reproduce? I use the Org layer and I don’t get this behaviour when I open an Org file. I do get some errors in the echo area, but they seem unrelated to your issue.

**Comment by Atrifex** [comment]
I have this in my user-config(). If you want to make a copy of yours and try placing this in there or if you see something blatantly wrong in there. I am not sure.

```
  (with-eval-after-load 'org
    (setq org-deadline-warning-days 14)
    (global-set-key (kbd "C-c c") 'org-capture)

    ;; I like on sequence, it might be helpful to change to multiple in the future
    (setq org-todo-keywords
          '((sequence
             "TODO(t)"
             "NEXT(n)" 
             "WAITING(w)"
             "SOMEDAY(.)"
             "MEET(m)"
             "BUG(b)"
             "READ(r)"
             "TODELEGATE(-)"
             "DELEGATED(d)"
             "|" "DONE(x!)" "FIXED(f)" "CANCELLED(c@)")))

    ;; catch invisible edits
    (setq org-catch-invisible-edits t)

    ;; stuff for GTD
    (setq org-agenda-custom-commands
          '(("W" "Weekly Review"
             ((agenda "" ((org-agenda-span 7)
                          ;;(org-agenda-start-day "-7d")
                          ;;(org-agenda-entry-types '(:timestamp))
                          (org-agenda-show-log t)))
              ;; (stuck "")
              (todo "TODO")  ;; review what is next
              (todo "WAITING") ;; projects we are waiting on
              (tags "PROJECT") ;; review all projects
              (tags "SOMEDAY") ;; review someday/maybe items
              ))

            ("D" "Daily review"
             ((agenda "" ((org-agenda-ndays 1)))
              (tags "URGENT") ;; review all projects
              (todo "TODO")
              ;; (todo "DELEGATED|WAITING") ;; projects we are waiting on
              ;; (stuck "")
              ))

            ("u" "Urgent Tasks"
             ((tags "URGENT") ;; review all urgent tasks
              ))
            )
          )

    (setq org-tags-exclude-from-inheritance '("@project"))

    (setq org-agenda-files (list "~/Documents/notes/personal/tasks.org"))


    ;; don't clutter headings with clock entries
    (setq org-log-into-drawer "LOGBOOK")
    (setq org-clock-into-drawer 1)
    )

```

**Comment by LemmingAvalanche** [comment]
I just copied over your `user-config()`. I still don’t that “garbage behaviour” when I open an org file. I tried with you `* Exporting to other formats` example.

**Comment by Atrifex** [comment]
Okay, well then i have no clue what could be causing it. I will look some more at the .spacemacs file and go through the .emacs.d folder to see if there are any other things and I will post back here if I find anything.

**Comment by LemmingAvalanche** [comment]
Please also run `spacemacs/describe-system-info` and paste the results here. ☺

**Comment by Atrifex** [comment]
Oh I think I found something! I used grep on the .emacs.d folder for CHANGED and this is what shows up:
<img width="1200" alt="screen shot 2016-11-06 at 3 01 45 pm" src="https://cloud.githubusercontent.com/assets/12935724/20041622/fd4a239a-a431-11e6-8e29-26e1cd13c95d.png">

**Comment by Atrifex** [comment]
This is the system info:

#### System Info :computer:
- OS: darwin
- Emacs: 25.1.1
- Spacemacs: 0.200.5
- Spacemacs branch: master (rev. 664ba6a)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(asm
 (auto-completion :variables auto-completion-enable-snippets-in-popup t auto-completion-return-key-behavior nil auto-completion-tab-key-behavior 'cycle auto-completion-private-snippets-directory "~/.spacemacs.d/snippets/" auto-completion-enable-help-tooltip 'manual :disabled-for org erc)
 bibtex
 (c-c++ :variables c-c++-enable-clang-support t)
 clojure command-log csharp csv dash django elfeed emacs-lisp emoji erc ess
 (evil-snipe :variables evil-snipe-enable-alternate-f-and-t-behaviors t)
 extra-langs games git github gtags haskell html
 (ibuffer :variables ibuffer-group-buffers-by nil)
 ipython-notebook javascript latex lua markdown
 (org :variables org-enable-bootstrap-support t)
 pdf-tools php
 (python :variables python-test-runner 'pytest)
 (ranger :variables ranger-override-dired t)
 react ruby rust search-engine
 (semantic :disabled-for emacs-lisp)
 (shell :variables shell-default-shell 'eshell)
 shell-scripts
 (spell-checking :variables spell-checking-enable-by-default t)
 spotify
 (syntax-checking :variables syntax-checking-enable-by-default nil)
 systemd typography
 (version-control :variables version-control-diff-tool 'diff-hl)
 vimscript yaml helm)
```
- System configuration features: JPEG RSVG IMAGEMAGICK NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES

**Comment by Atrifex** [comment]
I just realized that it does this for every file. Its just when the first file is opened however.

**Comment by TheBB** [comment]
How are you opening the files?

**Comment by Atrifex** [comment]
I am opening it from my most recently opened section.

**Comment by Atrifex** [comment]
Very interesting. Based on your question, I tried a couple of different ways to open the file. 

If I click on a file to open it in the spacemacs title screen, then it has the problem.
If i do <SPC> f r and then open the same file, then it consistently does not have this issue.

**Comment by Atrifex** [comment]
Here is a link to a video that I made showing the issue:

https://www.dropbox.com/s/3yjz2uuql6oakga/Spacmacs%20issue%20%237674.mov?dl=0

**Comment by TheBB** [comment]
Yep, I thought so. Duplicate of https://github.com/syl20bnr/spacemacs/issues/6765

**Comment by Atrifex** [comment]
Thank you!
